virttest/qemu_monitor.py

```diff
@@ -1258,6 +1258,29 @@ def get_migrate_cache_size(self):
         value = cache_size_info.split(":")[1].split()[0].strip()
         return value
 
+    def set_migrate_parameter(self, parameter, value):
+        """
+        Set parameters of migrate.
+
+        :param parameter: the parameter which need to set
+        :param value: the value of parameter
+        """
+        cmd = "migrate_set_parameter"
+        self.verify_supported_cmd(cmd)
+        cmd += " %s %s" % (parameter, value)
+        return self.cmd(cmd)
```

Comment:
Yes, I agree. But, I think they are all ok, just keep the coding style in the framework, like other functions